apps/src/applab/applab.js

```diff
@@ -2018,15 +2022,14 @@ Applab.container = function (opts) {
 };
 
 Applab.write = function (opts) {
-  // TODO: cpirich: may need to update param name
-  apiValidateType(opts, 'write', 'html', opts.html, 'uistring');
+  apiValidateType(opts, 'write', 'text', opts.html, 'uistring');
   return Applab.container(opts);
 };
 
 Applab.button = function (opts) {
   var divApplab = document.getElementById('divApplab');
 
-  // TODO: cpirich: may need to update param name
+  // PARAMNAME: button: id vs. buttonId
```

Comment:
does the PARAMNAME comment indicate that one param or the other (first or second) is the desired parameter name?